packages/debug-utils/index.js

```diff
@@ -530,13 +530,16 @@ var DebugUtils = {
     }
   },
 
-  formatInstruction: function (traceIndex, traceLength, instruction) {
+  formatCurrentInstruction: function (instruction) {
+    const pc = this.formatPC(instruction.pc);
+    const formattedInstruction = this.formatInstruction(instruction);
+    return (
+      truffleColors.mint("-> " + formattedInstruction + pc )
```

Comment:
So, having tried this... I don't think we want to apply the mint color to the whole thing.  I think the PC value should be left uncolored.

I'm not sure what to do about the arrow.  Probably make it a different color than the mint to make it stand out?

Reply:
change to
```
"-> " + truffleColors.mint(formattedInstruction) + pc
```